From 1d2250640065f5f78178ecae32f37ec24eebe442 Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Wed, 25 Jun 2014 10:02:38 +0100
Subject: [PATCH] mm: Rearrange zone fields into read-only, page alloc,
 statistics and page reclaim lines -prep

References: VM Performance
Patch-mainline: Never, unnecessary

This is a preparation patch to make a later patch easier to backport
from upstream.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 include/linux/mmzone.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/mmzone.h b/include/linux/mmzone.h
index 3e02545..4899050 100644
--- a/include/linux/mmzone.h
+++ b/include/linux/mmzone.h
@@ -397,6 +397,9 @@ struct zone {
 	spinlock_t		lru_lock;
 	struct lruvec		lruvec;
 
+	/* Evictions & activations on the inactive file list */
+	atomic_long_t		inactive_age;
+
 	unsigned long		pages_scanned;	   /* since last reclaim */
 	unsigned long		flags;		   /* zone flags, see below */
 
